backend/build.gradle.kts

```diff
@@ -1,6 +1,6 @@
 plugins {
     `maven-publish`
-    kotlin("jvm") version "1.7.10"
+    kotlin("jvm") version "2.0.0"
```

Comment:
```suggestion
    kotlin("jvm") version "1.8.0"
```

Reply:
Sorry I was reviewing this while tired last night and thought this was bumping the project version to 2.0.0 rather than Kotlin. Happy for you to revert this but also happy to merge!

Reply:
All good, you can merge. 👍